### PR TITLE
Fixes copyright, release and links in docs

### PR DESCRIPTION
**Pull request by codefromthecrypt**

### CODE REVIEW

File: doc/src/sphinx/conf.py [hunk 1]
```diff
@@ -36,18 +36,19 @@
 html_style = 'zipkin.css'
 
 project = u'Zipkin'
-copyright = u'2014 Twitter, Inc'
+copyright = u'2015 OpenZipkin contributors'
```
Comment (codefromthecrypt) [inline doc/src/sphinx/conf.py:39]: square is a rectangle, but not every rectangle is a square

File: doc/src/sphinx/conf.py [hunk 2]
```diff
@@ -36,18 +36,19 @@
...
 htmlhelp_basename = "zipkin"
+release = u'1.2.0-SNAPSHOT'
```
Comment (codefromthecrypt) [inline doc/src/sphinx/conf.py:41]: forgot this last time

**Comment by spencergibb** [comment]
:+1:

**Comment by codefromthecrypt** [comment]
http://openzipkin.github.io/zipkin/

**Pull request merged**